packages/components/src/styles/components/app-frame.scss

```diff
@@ -24,6 +29,10 @@
 .hds-app-frame__header {
   z-index: 7;
   grid-area: header;
+
+  &:has(.hds-app-header) {
+    min-height: var(--token-app-header-height); // reserve space for fixed header
```

Comment:
The AppHeader has a fixed position so this wrapper element collapses. I added the height to this wrapper so that content on the page wouldn't be overlapped by the fixed position AppHeader.

Reply:
**With the min-height:** 
_(all main content including breadcrumbs can be seen)_
<img width="1193" alt="image" src="https://github.com/user-attachments/assets/53228c6b-6dd2-4364-b734-f1b182b31a9d">

**With min-height commented out:**
_(main content is overlapped by the AppHeader so breadcrumbs are hidden)_
<img width="1190" alt="image" src="https://github.com/user-attachments/assets/ca5d797c-ad72-4618-b498-081b9fb2da5d">

Reply:
Replaced with adding `position: sticky` to AppFrame Header